libs/shared/data/src/lib/form-builder.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { FormlyFieldConfig, FormlyFormOptions } from '@ngx-formly/core';
+
+export interface IFormData {
```

Comment:
I'm not a fan of the `I` prefix for interfaces, if its done in other interfaces keep it but if not remove it

Reply:
Me either. The thing is that in a bunch of cases, we need Dto class and exactly the same interface for the FE. So, I had to add it.